src/index.js

```diff
@@ -1,20 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import Signup from './components/signup';
-import reportWebVitals from './reportWebVitals';
+import App from './App';
+import { BrowserRouter } from "react-router-dom"
 import 'bootstrap/dist/css/bootstrap.css';
 import 'font-awesome/css/font-awesome.css';
```

Comment:
Are you using font-awesome styles somewhere in your app?

Reply:
Not yet Sir